src/__tests__/componentHelpers/FilterLanguages.tsx

```diff
@@ -17,9 +17,7 @@ export const FilterLanguagesCommonTests = (
   });
 
   test("Language Filters shows all available languages", () => {
-    fireEvent.click(
-      screen.getAllByRole("button", { name: "Filter Languages" })[0]
-    );
+    fireEvent.click(screen.getByRole("button", { name: "Filter Languages" }));
```

Comment:
How did this just work?  Wasn't there two components that were mounted?  How could both the previous test and this test pass with no other changes?

Reply:
If the "unmount the unseen component" task is best separated out into a different ticket, then let's keep the `getAllByRole` tests and file that ticket.

Reply:
The changes were made in Search.tsx to use states instead of Chakra's built in responsive styling since there is currently no way to test Chakra's styling.